appveyor.yml

```diff
@@ -16,10 +16,10 @@ environment:
     - ruby_version: "21"
```

Comment:
i.e. I'm not sure what a local run is--I just mindlessly replaced 22 with 23. Or did you mean add a local run for 22 as well?